cluster/ci/config/prow/config.yaml

```diff
@@ -355,15 +355,28 @@ presubmits:
     rerun_command: "/test crio"
     trigger: "((?m)^/test crio,?(\\s+|$))"
 
-    # this job along with test_branch_origin_service_catalog will be deleted once the new catalog job is stable
-  - name: test_pull_request_origin_service_catalog
+  - name: test_pull_request_origin_service_catalog_39
     agent: jenkins
     labels:
       master: ci.openshift.redhat.com
     always_run: false
-    # TEMPORARY until this job is fixed to work
+    branches:
+    - release-3.9
+    # skip_report line will be deleted once testing is completed
```

Comment:
This is understood and doesn't warrant a comment

Reply:
Mostly a note to myself, but I can remove it.